flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/catalog/PostgresCatalog.java

```diff
@@ -231,13 +231,19 @@ public CatalogBaseTable getTable(ObjectPath tablePath) throws TableNotExistExcep
 	}
 
 	// Postgres jdbc driver maps several alias to real type, we use real type rather than alias:
+	// serial2 <=> int2
+	// serial4 <=> serial
+	// serial8 <=> bigserial
+	// smallserial <=> int2
```

Comment:
```suggestion
	// serial2 <=> int2
	// smallserial <=> int2
	// serial4 <=> serial
	// serial8 <=> bigserial
```
How about bubbling up "// smallserial <=> int2 " to make them look better?